packages/web/src/components/notification/Notification/RepostOfRepostNotification.tsx

```diff
@@ -0,0 +1,106 @@
+import { isMobile } from 'utils/clientUtil'
+import { useSelector } from 'utils/reducer'
+
+import { EntityLink, useGoToEntity } from './components/EntityLink'
+import { NotificationBody } from './components/NotificationBody'
+import { NotificationFooter } from './components/NotificationFooter'
+import { NotificationHeader } from './components/NotificationHeader'
+import { NotificationTile } from './components/NotificationTile'
+import { OthersLink } from './components/OthersLink'
+import { UserNameLink } from './components/UserNameLink'
+import { UserProfilePictureList } from './components/UserProfilePictureList'
+import { IconRepost } from './components/icons'
+import { entityToUserListEntity, USER_LENGTH_LIMIT } from './utils'
+const { getNotificationEntity, getNotificationUsers } = notificationsSelectors
+
+const messages = {
+  reposted: 'reposted your repost of'
+}
+
+type RepostOfRepostNotificationProps = {
+  notification: RepostOfRepostNotificationType
+}
+
+export const RepostOfRepostNotification = (
+  props: RepostOfRepostNotificationProps
+) => {
+  const { notification } = props
+  const { id, userIds, entityType, timeLabel, isViewed } = notification
+  const users = useSelector((state) =>
+    getNotificationUsers(state, notification, USER_LENGTH_LIMIT)
+  )
+  const firstUser = users?.[0]
+  const otherUsersCount = userIds.length - 1
+  const isMultiUser = userIds.length > 1
+
+  const entity = useSelector((state) =>
+    getNotificationEntity(state, notification)
+  )
+
+  const dispatch = useDispatch()
+
+  const handleGoToEntity = useGoToEntity(entity, entityType)
+
+  const handleClick: MouseEventHandler = useCallback(
+    (event) => {
+      if (isMultiUser) {
+        dispatch(
+          setUserListUsers({
+            userListType: UserListType.NOTIFICATION,
+            entityType: entityToUserListEntity[entityType],
+            id: id as unknown as number
+          })
+        )
+        if (isMobile()) {
+          dispatch(push(`notification/${id}/users`))
+        } else {
+          dispatch(openUserListModal(true))
+        }
+      } else {
+        handleGoToEntity(event)
+      }
+    },
+    [isMultiUser, dispatch, entityType, id, handleGoToEntity]
+  )
+
+  if (!users || !firstUser || !entity) return null
+
+  return (
+    <NotificationTile
+      notification={notification}
+      onClick={handleClick}
+      disableClosePanel={otherUsersCount > 0}
+    >
+      <NotificationHeader icon={<IconRepost />}>
+        <UserProfilePictureList
+          users={users}
+          totalUserCount={userIds.length}
+          stopPropagation
+        />
+      </NotificationHeader>
+      <NotificationBody>
+        <UserNameLink user={firstUser} notification={notification} />{' '}
+        {otherUsersCount > 0 ? (
+          <OthersLink othersCount={otherUsersCount} onClick={handleClick} />
+        ) : null}{' '}
+        {messages.reposted} {entityType.toLowerCase()}{' '}
```

Comment:
tiny nit: i think you may need to either hard code a {' '} space here or add a space to the messages.reposted value

Reply:
just double checked and i think we are good?